src/librustdoc/html/static/css/themes/light.css

```diff
@@ -43,10 +43,6 @@ pre, .rustdoc.source .example-wrap {
 	scrollbar-color: rgba(36, 37, 39, 0.6) #d9d9d9;
 }
 
-.rust-logo {
```

Comment:
In this case, it's to force other themes to have this rule.

Reply:
What do you mean by "force"?

Reply:
All the CSS rules in `light.css` must be present in the other themes, otherwise rustdoc will throw an error.